resources/factions/sweden_1980.json

```diff
@@ -0,0 +1,51 @@
+{
+  "country": "Sweden",
+  "name": "Sweden 1980",
+  "authors": "Turbolious",
+  "description": "<p>Sweden in the 80s.</p><p>Note : Limited A2A capabilities.</p>",
+  "locales": [
+    "sv_SE"
+  ],
+  "aircrafts": [
```

Comment:
I'm missing a Draken in this section 😝

Reply:
If i keep learning new stuff, who knows. I do know i can just pull it in from War thunder and make a mod out of it. I would just have to research how they make the FC3 mods lolol